yarn.lock

```diff
@@ -24450,6 +24457,18 @@ react-focus-lock@^2.9.2:
     use-callback-ref "^1.3.0"
     use-sidecar "^1.1.2"
 
+react-focus-lock@^2.9.4:
+  version "2.9.5"
+  resolved "https://registry.yarnpkg.com/react-focus-lock/-/react-focus-lock-2.9.5.tgz#8a82f4f0128cccc27b9e77a4472e8a22f1b52189"
+  integrity sha512-h6vrdgUbsH2HeD5I7I3Cx1PPrmwGuKYICS+kB9m+32X/9xHRrAbxgvaBpG7BFBN9h3tO+C3qX1QAVESmi4CiIA==
+  dependencies:
+    "@babel/runtime" "^7.0.0"
+    focus-lock "^0.11.6"
+    prop-types "^15.6.2"
+    react-clientside-effect "^1.2.6"
+    use-callback-ref "^1.3.0"
+    use-sidecar "^1.1.2"
+
 react-focus-on@^3.7.0:
```

Comment:
@cee-chen would you mind to bump kibana `react-focus-on` dependency as well, so we have dependencies aligned between Kibana and EUI?

Reply:
Great catch on this, thank you @patrykkopycinski! Resolved in https://github.com/elastic/kibana/pull/161716/commits/4042a90cb422dc3151df80a7edc5334d2500b3c4.

Just curious, do you mind clarifying why Kibana needs to specify `react-focus-on` as a dependency at all? As far as I can tell there's no imports or calls to it from within Kibana, just from EUI itself.

Reply:
I think we needed that to resolve some peer dependencies, but now I think we can remove it from Kibana package.json

Reply:
Huzzah! My favorite kind of cleanup is the kind where we get to delete stuff 🎉

Reply:
https://github.com/elastic/kibana/pull/161716/commits/9a33389d24895245299cdab4c7f4c935b0b29c12